lib/java/com/google/android/material/bottomsheet/BottomSheetBehavior.java

```diff
@@ -1867,6 +1872,16 @@ public boolean shouldSkipSmoothAnimation() {
     return true;
   }
 
+  /**
+   * Checks whether hiding gestures should be enabled while {@code isHideable} is set to true.
+   *
+   * @hide
+   */
+  @RestrictTo(LIBRARY_GROUP)
+  public boolean shouldEnableHidingGestures() {
```

Comment:
isHideableWhenDragging()? To be consistent with isHideable() and shouldSkipHalfExpandedStateWhenDragging()?

Reply:
Sure.